chrome/extension/devtools.js

```diff
@@ -6,6 +6,8 @@ const messenger = new Messenger();
 
 let connection;
 
+let styleManagerOpened = false;
```

Comment:
This needs to reset whenever setupConnection is called

Reply:
(or else it will be stale if the page changes)

Reply:
Actually, it probably can just always tell the bar to open up

Reply:
And the bar can choose to ignore it if it wants when its already open

Reply:
I added this bit because it was injecting multiple on the same page on top of itself - there has to be some logic to prevent the inline script from being injected multiple times. I can reset it when setupConnection is called.

Reply:
It has logic in there to early return if it detects the window object already exists

Reply:
Well then maybe logic is faulty? I didn't look at it that carefully but it 100% was injecting it many times. I'll try debugging that logic

Reply:
Possible! It does inject it multiple times but that should be fine because there's no side affects after the first time.

Reply:
Ok gonna take this bit out since we confirmed its me fixing a bug that should be fixed somewhere else